spec/models/questionable/question_spec.rb

```diff
@@ -26,20 +26,24 @@ module Questionable
       end
 
       it 'returns false if input_type is not string' do
-        question.input_type = 'date'
+        question.input_type = Questionable::Question::InputTypes::DATE
         expect(question).not_to be_string
       end
     end
 
-    %w(checkboxes multiselect radio select date).each do |kind|
+    [Questionable::Question::InputTypes::CHECKBOXES,
+     Questionable::Question::InputTypes::DATE,
+     Questionable::Question::InputTypes::MULTISELECT,
+     Questionable::Question::InputTypes::RADIO,
+     Questionable::Question::InputTypes::SELECT].each do |kind|
```

Comment:
@ascoppa there's gotta be a better way to do this right? Feels unnecessarily repetitive and unsure how to include variables from a module in a spec.

Reply:
```(Questionable::Question::InputTypes:ALL -  Questionable::Question::InputTypes::STRING).each do |kind|```

Reply:
Super cool 👍